methods/methods_release2.md

```diff
@@ -93,49 +95,262 @@ Demultiplexing, feature-table generation, fragment insertion, and taxonomic prof
+done`
 
-#### Taxonomic profiling
-* Process: Feature-classifier sklearn 
-* Parameters: Using the UNITE 8 release as a reference
+#### Denoising
+`deblur workflow \
+  --seqs-fp emp500_its_demux_trimmed_merged_files/ \
+  --output-dir emp500_its_deblur/ \
+  --trim-length 150 \
+  --threads-per-sample 0 \
+  --overwrite`
 
-#### OTU clustering
-* Process: Closed-reference OTU picking 
-* Parameters: 97% sequence similarity threshold, using the UNITE 8 release as a reference
+Import denoised sequences and feature-table into QIIME 2
+
+`qiime tools import \
+  --input-path emp500_its_deblur/all.biom \
+  --type 'FeatureTable[Frequency]' \
+  --input-format BIOMV210Format \
+  --output-path emp500_its_deblur/all_biom.qza`
+
+`qiime tools import \
+  --input-path emp500_its_deblur/all.seqs.fa \
+  --output-path emp500_its_deblur/all_seqs.qza \
+  --type 'FeatureData[Sequence]'`
+  
+At this point denoised feature-tables and sequences from each sequencing lane were merged.
 
+#### Merging tables:
+`qiime feature-table merge \
+  --i-tables emp500_its_deblur_prep_8702_biom.qza \
+  --i-tables emp500_its_deblur_prep_9961_biom.qza \
+  --i-tables emp500_its_deblur_prep_9962_biom.qza \
+  --i-tables emp500_its_deblur_prep_9963_biom.qza \
+  --i-tables emp500_its_deblur_prep_9964_biom.qza \
+  --p-overlap-method 'sum' \
+  --o-merged-table emp500_its_deblur_merged_biom.qza`
+
+`qiime feature-table merge-seqs \
+  --i-data emp500_its_deblur_prep_8702_seqs.qza \
+  --i-data emp500_its_deblur_prep_9961_seqs.qza \
+  --i-data emp500_its_deblur_prep_9962_seqs.qza \
+  --i-data emp500_its_deblur_prep_9963_seqs.qza \
+  --i-data emp500_its_deblur_prep_9964_seqs.qza \
+  --o-merged-data emp500_its_deblur_merged_seqs.qza`
+
+#### Taxonomic profiling:
+`qiime feature-classifier classify-sklearn \
+  --i-reads emp500_its_deblur_merged_seqs.qza \
+  --i-classifier unite9_2022.10.27_dynamic.qza \
+  --o-classification emp500_its_deblur_merged_seqs_taxonomy_unite9.qza`
+
+#### Filtering singleton features on a per-sample basis
+`qiime feature-table filter-features \
+  --i-table emp500_its_deblur_merged_biom.qza \
+  --p-min-samples 2 \
+  --o-filtered-table emp500_its_deblur_merged_biom_noSingletons.qza`
+  
+#### Filtering control samples
+`qiime feature-table filter-samples \
+  --i-table emp500_its_deblur_merged_biom_noSingletons.qza \
+  --m-metadata-file emp500_metadata_basic.txt \
+  --p-where 'empo_1 != "Control"' \
+  --o-filtered-table emp500_its_deblur_merged_biom_noSingletons_noControls.qza` 
+
+#### Filtering samples with low read counts
+`qiime feature-table filter-samples \
+  --i-table emp500_its_deblur_merged_biom_noSingletons_noControls.qza \
+  --p-min-frequency 500 \
+  --o-filtered-table emp500_its_deblur_merged_biom_noSingletons_noControls_min500.qza` 
+
+#### Estimate beta-diversity
+`qiime deicode rpca \
+  --i-table emp500_its_deblur_merged_biom_noSingletons_noControls_min500.qza \
+  --p-min-feature-count 0 \
+  --p-min-sample-count 0 \
```

Comment:
18S processing also used `--p-min-feature-frequency 10 \`, is that important?

Reply:
Thanks for catching that! I've updated the ITS code to include the filter as we used it.